ludwig/schema/hyperopt/search_algorithm.py

```diff
@@ -54,3 +81,675 @@ def _jsonschema_type_mapping(self):
         },
         default_factory=load_default,
     )
+
+
+@DeveloperAPI
+@register_search_algorithm("random", random_state_field="random_state")
+@register_search_algorithm("variant_generator")
```

Comment:
Do we also need to add `random_state_field="random_state"` to the `variant_generator` entry in the registry?

Reply:
Good catch!